exporter/metric/metric.go

```diff
@@ -74,6 +74,42 @@ type metricExporter struct {
 	startTime time.Time
 }
 
+
+// TODO: put all constants into a separate file in the same package
```

Comment:
Most likely these constants will eventually be included in the Go SDK and we can just reference those values directly, but for now putting them in the exporter is fine (but yes you should move them to another file)

Reply:
+1. For the sake of easier removal after Go SDK's introduction of auto detector, moving relevant contents into another file in this PR sounds better.